### PR TITLE
[5.7.x] update the plugin version #686

### PR DESCRIPTION
**Pull request by btonokit**
Please review #715

The changes other than the version upgrade are as follows:

- The `com.google.code:maven-license-plugin` has been changed to `com.mycila:license-maven-plugin` as in https://github.com/terasolunaorg/terasoluna-gfw/issues/600.
- The `build-helper-maven-plugin` didn't seem to be used so I removed it.
- Fixed the variable name of the plugin version to be `groupId.artifactId.pugin.version`.
https://github.com/terasolunaorg/terasoluna-tourreservation/pull/716/commits/60dc18b78de6c833a2612e13a97deb842db8d395

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -51,43 +51,10 @@
   <build>
     <pluginManagement>
       <plugins>
-        <plugin>
-          <groupId>org.codehaus.mojo</groupId>
-          <artifactId>build-helper-maven-plugin</artifactId>
-          <version>${org.codehaus.mojo.build-helper-maven-plugi.version}</version>
```
Comment (yoshikawaa) [inline pom.xml:57]: Why make unnecessary changes from the blank project definition?

The blank project contains `build-helper-maven-plugin`.
https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/d3b8feb53805dbe5c7119c3ea7717d899ed488bf/pom.xml#L57-L89

Comment (btonokit) [reply pom.xml:57]: build-helper-maven-plugin doesn't seem to be used in this application.
`maven-failsafe-plugin` is also included in the blank project, but not in this application.
https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/d3b8feb53805dbe5c7119c3ea7717d899ed488bf/pom.xml#L40-L51
Therefore, I decided that it is not necessary to set unused plugins in this application according to the blank project, and removed `build-helper-maven-plugin`.

Comment (yoshikawaa) [reply pom.xml:57]: What do you think is the position of this application? If the position it as a sample application starting from a blank project, don't you think you should avoid unnecessary changes?
Making the sample application independent of the blank project means the growth of different branches.

Thanks.

Comment (btkatoufm) [reply pom.xml:57]: Many repositories, including private repositories, have been modified.
You should check again to make it a blank project compliant.

Comment (btonokit) [reply pom.xml:57]: `maven-failsafe-plugin` should have been set for this application as well for the blank project, but it seems that the reflection was missing.
I will take action to fit the blank project.
https://github.com/terasolunaorg/terasoluna-tourreservation/issues/686#issuecomment-891654903

Comment (btonokit) [reply pom.xml:57]: Modified each pom.xml to match multi-blank.
d5272d42f2615d97eea7abc9f2bf12a23ffd9399
